package.xml

```diff
@@ -1014,8 +1014,8 @@ Update to wrap gRPC C Core version 0.10.0
   </release>
   <release>
    <version>
-    <release>0.14.2</release>
-    <api>0.14.2</api>
+    <release>0.14.0</release>
```

Comment:
So `0.14.2` was substituted into the file because the release placeholder has already advanced to that. This is however the history section, so I need to re-freeze that section to what it was, when the file was previously released, which was `0.14.0`.